src/ER/PatientInfo.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const time = "09:57:25"
+const medic = "Medic 16"
+const MajorComplaint = "Chest pain. Mild headach"
+
+const PatientInfo = () => {
```

Comment:
I think we'll end up using Christian's implementation of this component, so I'd delete it and any references to it here (I think it's unused right now) so we don't have duplicates when we get both branches merged onto master.